scripts/badger/voter_processing_unlocks.py

```diff
@@ -0,0 +1,20 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
```

Comment:
lets incorp this into `voter_chores` as well as to not have redundant code

Reply:
resolved in bef27e9818c2f9d02ff5892f422df72e78f50250